vision_datasets/visual_object_grounding/operations.py

```diff
@@ -24,4 +24,6 @@ def process_labels(self, coco_ann, label: ImageLabelManifest):
 @StandAloneImageListGeneratorFactory.register(_DATA_TYPE)
 class Text2ImageRetrievalStandAloneImageListGenerator(GenerateStandAloneImageListBase):
     def _generate_label(self, label: VisualObjectGroundingLabelManifest, image: ImageDataManifest, manifest: DatasetManifest) -> dict:
-        return {'question': label.question, 'answer': label.answer, 'groundings': label.label_data['groundings']}
+        return {'question': label.question,
+                'answer': label.answer,
+                'grounding_metadata': {'groundings': label.label_data['groundings'], 'gt_answer': label.answer, 'size': [image.width, image.height]}}
```

Comment:
image size does not really belong to annotations.

Ideally, metric evaluator should be asking for it. User of the evaluator could get the image size info from dataset and pass it to evaluator, instead of taking it from annotation